src/custom/hooks/useRefetchPriceCallback.tsx

```diff
@@ -202,7 +190,9 @@ export function useRefetchQuoteCallback() {
 
         // Update quote
         updateQuote(quoteData)
```

Comment:
Do we ALWAY update the price? no matter if its worse or better?

Reply:
Addressed in the top comment